hugegraph-rocksdb/src/main/java/com/baidu/hugegraph/backend/store/rocksdb/RocksDBOptions.java

```diff
@@ -359,4 +359,28 @@ public static synchronized RocksDBOptions instance() {
                     disallowEmpty(),
                     false
             );
+
+    public static final ConfigOption<Long> MAX_MANIFEST_FILE_SIZE =
+            new ConfigOption<>(
+                    "rocksdb.max_manifest_file_size",
+                    "The max size of manifest file in bytes.",
+                    rangeInt(1L, Long.MAX_VALUE),
+                    104857600L
+            );
+
+    public static final ConfigOption<Boolean> SKIP_STATS_UPDATE_ON_DB_OPEN =
+            new ConfigOption<>(
+                    "rocksdb.skip_stats_update_on_db_open",
+                    "This flag allows us to not update statistics.",
```

Comment:
Whether to skip statistics update when opening the database, 
setting this flag true allows us to not update statistics.